src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/legalrepresentative/LegalRepresentativeRemissionDecisionRejectedPersonalisation.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.legalrepresentative;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.*;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.EmailNotificationPersonalisation;
+
+@Component
+public class LegalRepresentativeRemissionDecisionRejectedPersonalisation implements EmailNotificationPersonalisation {
+
+    private final String rejectedTemplateId;
+    private final String iaExUiFrontendUrl;
+
+    public LegalRepresentativeRemissionDecisionRejectedPersonalisation(
+        @Value("${govnotify.template.recordRemissionDecision.rejected.legalRep.email}") String rejectedTemplateId,
+        @Value("${iaExUiFrontendUrl}") String iaExUiFrontendUrl
+
```

Comment:
Same comments as above apply to this class.

Reply:
same fix